src/sbd-pacemaker.c

```diff
@@ -58,6 +58,38 @@
 
 #include "sbd.h"
 
+#ifndef HAVE_PE_NEW_WORKING_SET
+
+static void
+pe_reset_working_set(pe_working_set_t *data_set)
+{
+	if (data_set != NULL) {
+		cleanup_calculations(data_set);
+		set_working_set_defaults(data_set);
```

Comment:
cleanup_calculations() will call set_working_set_defaults(), and it's OK with NULL data_set, so you can just call cleanup_calculations()